R/admiraldev-package.R

```diff
@@ -24,5 +24,5 @@
 #' @importFrom tidyselect all_of contains vars_select
 #' @importFrom hms as_hms
 #' @importFrom lifecycle deprecate_warn deprecated deprecate_stop
-#' @importFrom admiral derive_vars_dy
+#' @importFrom admiral derive_vars_dy assert_character_scalar vars2chr filter_if
```

Comment:
WHy would `assert_character_scalar()` stay in {admiral}?

Reply:
it shouldn't i think this is holdover from when i was bringing things over.  i wanted to just bring over bare minimum and did this importfrom assert to just test some stuff out.  will remove.